### PR TITLE
Tinker Requests

### PR DESCRIPTION
**Pull request by j-jasik**
Added a method in the tinker controller that handles all requests.get()

### CODE REVIEW

File: tinker/admin/program_search/program_search_controller.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def get_programs_for_dropdown(self, return_dict_for_renaming=False):
         else:
             program_concentrations = []
 
-        response = requests.get(app.config['PROGRAMS_XML'])
+        response = self.tinker_requests(app.config['PROGRAMS_XML'])
+        # response = requests.get(app.config['PROGRAMS_XML'])
```
Comment (c-schwarze) [inline tinker/admin/program_search/program_search_controller.py]: can you remove all of the commented out code?

File: tinker/faculty_bios/forms.py [hunk 2]
```diff
@@ -55,7 +55,8 @@ def validate_username(form, field):
     username = field.data
     host = "http://wsapi.bethel.edu"
     path = "/username/" + username + "/roles"
-    req = requests.get(host + path)
+    req = FacultyBioController.tinker_requests(host + path)
```
Comment (c-schwarze) [inline tinker/faculty_bios/forms.py]: can you actually clean up this code? Make the host and path be set in one line.
`url = "http://wsapi.bethel.edu/username/%s/roles" % username

File: tinker/admin/program_search/program_search_controller.py (hunk 1 above)
Comment (j-jasik) [reply tinker/admin/program_search/program_search_controller.py]: Removed

File: tinker/faculty_bios/forms.py (hunk 2 above)
Comment (j-jasik) [reply tinker/faculty_bios/forms.py]: Done

**Pull request merged**